### ISSUE TITLE
Evaluate iambic as a new user using a more moderate read-only type spoke role

### ISSUE THREAD

**Issue by smoy** [noqdev/iambic#323]
**Is your feature request related to a problem? Please describe.**
It would be great if somehow I could evaluate iambic as a new user, using a more moderate read-only type spoke role, vs something that can create access keys, revoke permissions, etc. right off the bat.

**Describe the solution you'd like**

The permission should be much more restricted than iam:* and sso:*

```
              - Effect: Allow
                 Action:
                   - identitystore:Describe*
                   - identitystore:Get*
                   - identitystore:List*
                   - organizations:describe*
                   - organizations:list*
                   - iam:Get*
                   - iam:List*
                   - sso:Describe*
                   - sso:Get*
                   - sso:List*
                   - sso:Search*
                 Resource:
                   - '*'
```

**Describe alternatives you've considered**
Supply my own SpokeRole to use in configuration

**Additional context**
```
  accounts:
    - account_id: 'REDACTED'
      account_name: iambic_test_spoke_account_2
      iambic_managed: read_and_write
      org_id: 'REDACTED'
      spoke_role_arn: arn:aws:iam::REDACTED:role/IambicSpokeRoleReadOnly
```

**Comment by smoy** [comment]
https://github.com/noqdev/iambic/pull/317 should tackle this

**Comment by smoy** [comment]
We've merged #317 , so it's available on the main branch. If anyone wants to try it out, clone the repo directory. This will be included in the next pypi build 0.3.0+. I will close this issue when we publish the build on PyPI.

**Comment by smoy** [comment]
This is now included in pypi release 0.3.1. Marking this issue as closed.
